app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,19 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], html: {class: "form-horizontal"} do |f|
```

Comment:
You should read `bootstrap_form_for` docs. Horizontal forms are rather labeled as `layout: :horizontal` check that out.